torch/_inductor/lowering.py

```diff
@@ -2113,6 +2111,12 @@ def is_aligned_backward(x):
     sdpa_constraint,
     warn=False,
 )
+make_fallback(
+    torch.ops.aten._flash_attention_forward.default, sdpa_constraint, warn=False
+)
+make_fallback(
+    torch.ops.aten._flash_attention_backward.default, sdpa_constraint, warn=False
+)
 make_fallback(torch.ops.aten._efficient_attention_forward.default)
```

Comment:
Also could you add the constraints for this one as well that was a mistake in my last PR and should have addded